### DISCUSSION TITLE
videoConstraints update on the go

### DISCUSSION THREAD

**Discussion by AntonDr** [mebjas/html5-qrcode#329] [General]
Is there any ability to update videoConstraints when camera is open?

**Comment by mebjas** [comment]
It's supported on `Html5Qrcode` (undocumented and beta) -- [src]()

```js
// qr code is running
html5Qrcode.applyVideoConstraints(videoConstaints).then(/** success callback */);
```

Do you need this for `Html5QrcodeScanner`?

Also can you state your usecase?

**Reply by AntonDr** [reply]
Initially I need video zoom capability. videoConstraints: { zoom: zoomValue }. Is there any other way to do that?

No, for Html5Qrcode

**Reply by mebjas** [reply]
>  for Html5Qrcode

You can use the link I shared above

```js
// qr code is running
html5Qrcode.applyVideoConstraints(videoConstaints).then(/** success callback */);
```

**Reply by AntonDr** [reply]
applyVideoConstraints({ zoom: zv }) should works?

**Comment by ROBERT-MCDOWELL** [comment]
it would be worth indeed.
